OtripleS.Web.Api.Tests.Unit/Services/TeacherServiceTests/TeacherServiceTests.Exceptions.Retrieve.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
We already have these tests. Why did we created them again?

Reply:
@viralpandya  I added this file last week, where it was not exists in master repository.

Reply:
When you pull from **Repo:**Hassanhabib/OtripleS and **Branch:**master, then it should have given you conflict. 

These test cases are not associated with the service code in TDD approach. Also, this code is already in the repo. So you may want to delete them or else it may create conflict or duplicate code.

Reply:
TeacherServiceTests.Exceptions.Retrieve is removed.